### PR TITLE
BUGFIX: BB-1451, BUGFIX: BB-1452, BUGFIX: BB-1453 Fix pivot totals menu

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3634/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2906/ : SUCCESS in 13m 39s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3658/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2921/ : SUCCESS in 6m 09s

File: src/components/core/pivotTable/tests/aggregationsMenuHelper.spec.ts [hunk 1]
```diff
@@ -135,14 +101,59 @@ describe('aggregationsMenuHelper', () => {
...
+
+            expect(aggregationsMenuHelper.getTotalsForAttributeHeader(totals, measures)).toEqual([
+                {
+                    type: 'sum',
+                    attributes: ['a1']
+                }
+            ]);
+        });
+
+        it('should return sum, min totals when totals are defined for only measure', () => {
```
Comment (strouhaljan) [inline src/components/core/pivotTable/tests/aggregationsMenuHelper.spec.ts]: How about: "... for single measure"?

File: src/components/core/pivotTable/tests/aggregationsMenuHelper.spec.ts [hunk 2]
```diff
@@ -135,14 +101,59 @@ describe('aggregationsMenuHelper', () => {
...
                     type: 'sum',
                     measureIdentifier: 'm2',
+                    attributeIdentifier: 'a2'
+                }
+            ];
+
+            expect(aggregationsMenuHelper.getTotalsForAttributeHeader(totals, measures)).toEqual([]);
+        });
+
+        it('should return sum total when totals are defined for all measures and same attribute', () => {
```
Comment (strouhaljan) [inline src/components/core/pivotTable/tests/aggregationsMenuHelper.spec.ts]: How about: "... and single attribute"?

File: src/components/core/pivotTable/aggregationsMenuHelper.ts [hunk 3]
```diff
@@ -2,37 +2,48 @@
...
-): AFM.TotalType[] => {
-    return AVAILABLE_TOTALS.filter((type) => {
-        const totalsOfType = totals.filter((total: AFM.ITotalItem) => total.type === type);
-        const totalsMeasureIdentifiers = totalsOfType.map((total: AFM.ITotalItem) => total.measureIdentifier);
-        return measureLocalIdentifiers.every((measureIdentfier: string) => {
-            return totalsMeasureIdentifiers.includes(measureIdentfier);
-        });
-    });
-};
+function getAttributeIntersection(totals: AFM.ITotalItem[], type: AFM.TotalType, measures: string[]) {
```
Comment (strouhaljan) [inline src/components/core/pivotTable/aggregationsMenuHelper.ts]: Please split this function in two separate functions.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3662/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2925/ : SUCCESS in 9m 25s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3666/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2929/ : SUCCESS in 13m 09s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1571/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3667/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/965/ : SUCCESS in 17m 14s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3670/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2932/ : SUCCESS in 2m 21s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1572/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3671/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/966/ : UNSTABLE in 21m 05s

**Comment by petrjanu** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-860.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3672/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2934/ : SUCCESS in 4m 43s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3673/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/967/ : FAILURE in 29m 37s

**Comment by petrjanu** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-860.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3675/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/968/ : SUCCESS in 21m 20s

**Comment by strouhaljan** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/797/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2940/ : SUCCESS in 3m 53s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1575/ : SUCCESS in 15m 25s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/794/ : SUCCESS in 18m 56s

**Pull request merged**